app/eventyay/config/settings.py

```diff
@@ -380,6 +380,9 @@ def instance_name(request):
     }
 }
 
+# JSON field support is available in PostgreSQL
+JSON_FIELD_AVAILABLE = DATABASES['default']['ENGINE'].split('.')[-1] == 'postgresql'
```

Comment:
We only supports PostgreSQL, meaning `JSONField` is always available. Don't add and don't depend on this setting.